integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6810,7 +6810,15 @@ msgstr ""
 "Sie haben nicht die nötige Berechtigung, um diese Seite zu veröffentlichen, "
 "aber Sie können Änderungen vornehmen und diese zur Überprüfung vorlegen."
 
-#: cms/views/pages/page_form_view.py:295
+#: cms/views/pages/page_form_view.py:302
+msgid ""
+"Page \"{}\" was saved as major version. The first version is not allowed to "
+"be a minor edit."
+msgstr ""
+"Seite \"{}\" wurde als NICHT-geringfügige Änderung gespeichert. Die erste "
```

Comment:
I'd stick to the English original "Seite wurde als große Änderung gespeichert."

Reply:
done!